.github/workflows/chart-release.yml

```diff
@@ -0,0 +1,32 @@
+name: Chart Release
+on:
+  release:
+    types: [published]
+
+jobs:
+  release:
+    permissions:
+      contents: write
+    runs-on: ubuntu-latest
+    steps:
+    
+      - name: Checkout
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+
+      - name: Configure Git
+        run: |
+          git config user.name "$GITHUB_ACTOR"
+          git config user.email "$GITHUB_ACTOR@users.noreply.github.com"
+
+      - name: Install Helm
+        uses: azure/setup-helm@v3
+
+      - name: Run chart-releaser
+        uses: helm/chart-releaser-action@v1.6.0
+        env:
+          CR_TOKEN: "${{ secrets.GITHUB_TOKEN }}"
+          CR_OWNER: telekom-mms
+          CR_GIT_REPO: trivy-dojo-report-operator
+          CR_RELEASE_NAME_TEMPLATE: "{{ .Version }}"
```

Comment:
If this runs on published releases, can we set this `.Version` to the tag?
We probably have to do this in the other files, too (with sed or similar).

Reply:
This depends on how the final release should look like. 

In my mind the release would look like this:

GitHub Release Name: 1.1.4
helm chart:  helm install myreleasename telekom-mms/trivy-dojo-report-operator --version 1.1.4
container image: ghcr.io/telekom-mms/trivy-dojo-operator:1.1.4

The container image gets its version tag from the github release name, which originally comes from the chart metadata

Reply:
Okay. So I would like to only have to once specify the version somewhere (preferably as a github release) and all other places should get it automatically.

Reply:
Got it.

Now I have a mechanism that updates all required files with the version of the release.
There is just one issue - I most likely need to squeeze it within the pre-release/draft creation, but I have not figured out a way to do this yet